FileRead&write.c

```diff
@@ -1,152 +0,0 @@
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include "debugmalloc.h"
-#include "structs.h"
-
-extern void ReadIngredientFile() {
-    FILE* F_file_Ingredients_csv = fopen("ingredients.csv", "r");
-    if (F_file_Ingredients_csv == NULL) {
-        perror("Failed to open ingredients.csv");
-        exit(1);
-    }
-    Ingredient I_tmpIngredient;
-
-   while(fscanf(F_file_Ingredients_csv, "%[^,],%[^,],%[^\n]\n",
-              I_tmpIngredient.c_Ingredient_ID ,
-              I_tmpIngredient.c_Ingredient_Name,
-              I_tmpIngredient.c_Unit) != EOF ){
-
-           
-    }
-    fclose(F_file_Ingredients_csv);
-}
-
-extern void ReadRecipeFile() {
-    FILE* F_file_Recipe_csv = fopen("recipes.csv", "r");
```

Comment:
This part is the same in every function.
You can make a separate fucntion:
FILE* OpenFile(char[] filename){
FILE* F_file = fopen(filename, "r");
    if (F_file_Ingredients_csv == NULL) {
        perror("Failed to open ingredients.csv");
        exit(1);
    }
    return F_file;
}

Reply:
I see, and that's make the code more transparent ?